tests/acceptance/api_test.go

```diff
@@ -1330,6 +1330,19 @@ func TestApi(t *testing.T) {
 			log.Println("Lease created. Waiting for account to be marked 'Leased'")
 			waitForAccountStatus(t, apiURL, accountID, "Leased")
 
+			// Check the lease LeaseStatusModifiedOn
+			resp := apiRequest(t, &apiRequestInput{
+				method: "GET",
+				url:    apiURL + fmt.Sprintf("/leases?principalId=test-user&accountId=%s", accountID),
+				json:   nil,
+			})
+
+			results := parseResponseArrayJSON(t, resp)
+			assert.Equal(t, 200, resp.StatusCode)
+			assert.Equal(t, 1, len(results), "one lease should be returned")
+			assert.Equal(t, "Active", results[0]["leaseStatus"])
+			assert.NotNil(t, results[0]["leaseStatusModifiedOn"])
```

Comment:
We're probably not doing this anywhere else, but do we want to add any different checks other than NotNil? Should it be formatted a certain way? Greater than today, etc.?

Reply:
@nathanagood I will add a check to verify leaseStatusModifiedOn date is equal to CreatedOn date.